tests/find.spec.ts

```diff
@@ -4,15 +4,45 @@ test('has a title and a list of accredited programmes', async ({ page }) => {
   await page.goto('/programmes')
   await expect(page.locator('h1')).toHaveText('List of accredited programmes')
   const courseLinks = await page.locator('div[role="list"] a')
-  expect(courseLinks).toHaveText(['Lime Course', 'Azure Course', 'Violet Course'])
+  expect(courseLinks).toHaveText([
+    'Becoming New Me Plus (BNM+)',
+    'Becoming New Me Plus (BNM+)',
+    'Becoming New Me Plus (BNM+)',
+    'Building Better Relationships (BBR)',
+    'Healthy Identity Intervention (HII)',
+    'Healthy Sex Programme (HSP)',
+    'Horizon',
+    'Identity Matters (IM)',
+    'Identity Matters (IM)',
+    'Kaizen',
+    'Kaizen',
+    'Kaizen',
+    'Living as New Me (LNM)',
+    'New Me Strengths (NMS)',
+    'New Me Strengths (NMS)',
+    'New Me Strengths (NMS)',
```

Comment:
I assume there is a reason some of these are duplicated

Reply:
Yup - we have multiple courses with the same name but different audiences